api/handlers.go

```diff
@@ -0,0 +1,25 @@
+package api
+
+import (
+	"encoding/json"
+	"github.com/ONSdigital/dp-sessions-api/session"
+	"github.com/ONSdigital/log.go/log"
+	"net/http"
+)
+
+func CreateNewSessionHandler() http.HandlerFunc {
```

Comment:
Name is slightly confusing - is it creating a new session or creating a new session handler? I would probably go with `CreateSessionHandlerFunc` - as this function is not the handler but returns the a http handler func. Nit picking but you asked for a review 😉